SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/BrokerManaging/BrokerJSONServiceProvider.swift

```diff
@@ -80,20 +80,20 @@ public extension BrokerStoring {
         }
 
         guard Self.shouldUpdate(incoming: broker.version, storedVersion: savedBroker.version) else {
-            Logger.dataBrokerProtection.log("False positive (changed eTag but same version): \(broker.url, privacy: .public)")
+            Logger.dataBrokerProtection.log("🧩 False positive (changed eTag but same version): \(broker.url, privacy: .public)")
```

Comment:
What's the reasoning behind the 🧩? We do have a few different loggers within DBP that we don't make great use of, it might be worth considering using a new one

Reply:
Ah just to make it easier to filter out Console.app (similar to the 👾 we have for pixels).

Reply:
Okay, I did wonder why we use 👾 tbh 😅